waggle-dance-core/src/test/java/com/hotels/bdp/waggledance/mapping/service/impl/StaticDatabaseMappingServiceTest.java

```diff
@@ -95,6 +99,24 @@ private MetaStoreMapping mockNewMapping(boolean isAvailable, AbstractMetaStore m
     return result;
   }
 
+  private FederatedMetaStore newFederatedInstanceWithClient(
+      String name,
+      String uri,
+      List<String> mappedDatabases,
+      boolean availableMapping) {
+    FederatedMetaStore newMetastore = newFederatedInstance(name, uri);
+    newMetastore.setMappedDatabases(mappedDatabases);
+    MetaStoreMapping newMapping = mockNewMapping(availableMapping, newMetastore);
+    when(metaStoreMappingFactory.newInstance(newMetastore)).thenReturn(newMapping);
+    when(newMapping.getClient()).thenReturn(federatedDatabaseClient);
+    try {
+      when(federatedDatabaseClient.get_all_databases()).thenReturn(mappedDatabases);
+    } catch (TException e) {
```

Comment:
why try catch? just throw we're in a test

Reply:
I thought that if it was in a `try`, then many tests wouldn't have to be changed to include `throws MetaException, TException` but it seems that I there weren't many to change anyway.

Reply:
Generally in a test, if something is wrong just throw and fail as it is probably a bug.  Perfectly fine to have `throws Exception` on every method. We want to fail as fast as possible. With you're try catch and printing test might fail and we would have to scroll to the logs to figure out why would be issue to miss.